apps/projects/package.json

```diff
@@ -29,6 +29,7 @@
     "@aragon/ui": "0.30.1",
     "@tps/ui": "^0.0.1",
     "apollo-boost": "^0.1.22",
+    "axios": "^0.18.0",
```

Comment:
I don't love importing this _just_ for the one line but if I'm the only person that feels that way it's not a big deal.

Reply:
The problem i was having with using native browser `fetch` is that it can send a preflight request to a backend to establish a connection (or for various other reasons i'm not aware of). The prefligth request is an `OPTIONS` request, which gives back a CORS error from infura 

Maybe our recent upgrades will fix this tho and then we can take it out and potentially not even write our own getter method